db/migrate/20160510173147_create_peer_reviews.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePeerReviews < ActiveRecord::Migration
+  def change
+    create_table :peer_reviews do |t|
+      t.references :result, index: true, foreign_key: true, null: false
+      t.integer :reviewer_id, null: false
```

Comment:
Ah, somehow I thought there was a way to specify the column name for `t.references`. It turns out not to matter, though, because we can specify the foreign key as a separate operation (`add_foreign_key`).

So you should be able to do `t.references :reviewer` and then later add a foreign key for the column `reviewer_id`. Same with reviewee.